### PR TITLE
Added logging (issue #233) and views (issue #257)

### PR DESCRIPTION
**Pull request by dimitri-yatsenko**
* Solved issue #233 as described 
* Solved issue #257 as described

### CODE REVIEW

**Comment by eywalker** [comment]
Please describe your PR.

File: datajoint/base_relation.py [hunk 1]
```diff
@@ -474,3 +488,58 @@ def table_name(self):
...
+            # copy constructor
+            self.database = arg.database
+            self._connection = arg._connection
+            self._definition = arg._definition
+            self._user = arg._user
+            return
+
+        self.database = database
+        self._connection = arg
+        self._definition = """    # job reservation table for `{database}`
```
Comment (eywalker) [inline datajoint/base_relation.py]: Table comment needs to be updated.

Comment (dimitri-yatsenko) [inline datajoint/base_relation.py]: fixed

File: datajoint/base_relation.py [hunk 2]
```diff
@@ -474,3 +488,58 @@ def table_name(self):
         return self._table_name
 
 
+class Log(BaseRelation):
+    """
+    A base relation with no definition. Allows reserving jobs
```
Comment (dimitri-yatsenko) [inline datajoint/base_relation.py]: fixed

File: datajoint/base_relation.py [hunk 3]
```diff
@@ -474,3 +489,59 @@ def table_name(self):
...
+            # copy constructor
+            self.database = arg.database
+            self._connection = arg._connection
+            self._definition = arg._definition
+            self._user = arg._user
+            return
+
+        self.database = database
+        self._connection = arg
+        self._definition = """    # even logging table for `{database}`
```
Comment (eywalker) [inline datajoint/base_relation.py]: typo: even should be event. Could you fix that while you are at it?

Comment (dimitri-yatsenko) [inline datajoint/base_relation.py]: fixed

**Pull request merged**